### PR TITLE
Overview of Rootkits

### LINKED ISSUE #5432: [Security] Overview of Rootkits
#### Proposed title of article
Overview of Rootkits

#### Proposed article introduction
Security and privacy have nearly become myths in this digital age of computers and technology. As much as data has become crucial, its protection has become even more valuable. Today, securing our data and information is our top priority. There are numerous ways for someone to get unauthorized access to our systems and steal the data. Rootkits are one such method.
 
Rootkits are malicious software that targets the operating system, firmware, or various system applications in order to gain root privileges. They have the ability to go undetected by the system and deploy various malware that allows them to steal data.

#### Key takeaways 
- Understand what are rootkits
- What makes them vulnerable
- How they affect our system
- How they can be used to improve existing systems
- Different types of rootkits

#### Article quality
This article provides a quick overview of rootkits and gets the reader started. It is intended for beginners and anyone interested in learning more about one of the many topics in cybersecurity. The types of rootkits, how they function, how to detect and avoid them, and so on should be able to help them comprehend the type of threat that exists that one might face in the future.

#### References
https://en.wikipedia.org/wiki/Rootkit
https://www.kaspersky.com/resource-center/definitions/what-is-rootkit

### PR DESCRIPTION
**Pull request by kvardhan10**
### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5432

### CODE REVIEW

**Comment by Kayere** [comment]
Plagiarism check done ✅

File: content/articles/overview-of-rootkits/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+#### Bootkits
+Bootkits are a type of kernel-mode rootkit that affects the system's booting mechanism. Rootkits of this type replace the original bootloader with the infected one. This allows the hackers to access the system's startup code. This launches the rootkit even before your computer's operating system is fully loaded, resulting in rootkit execution when the machine ultimately boots up.
+
+#### Firmware rootkit
+A firmware rootkit is a type of rootkit that exploits essential hardware such as routers, drives, and so on. It takes advantage of the firmware because it is unaffected by unit testing and security software. Before deploying a software update, the system is reviewed for quality and tested to maximize code coverage. During this stage, the majority of harmful files and codes are eliminated from the system. As a result, this type of rootkit latches onto the firmware and uses it as leverage.
+
+### Preventive measures
```
Comment (geoffrey45) [inline content/articles/overview-of-rootkits/index.md:35]: The points in this section sound important. Please convert each paragraph into a sub-heading with a description below it.

**Comment by kvardhan10** [comment]
@geoffrey45 Made the desired changes. Let me know if anything else is required

**Comment by geoffrey45** [comment]
Thank you @kvardhan10 for the amazing artilce.

**Comment by WanjaMIKE** [comment]
Merged #5920 into master

**Comment by hectorkambow** [comment]
@geoffrey45 

LEts please be sure we are careful with the front matter (author info) this article has another author that did not write the article and the payout was issued to him instead. 

cc @kvardhan10 

cc @francis966

**Pull request merged**